src/hotspot/share/prims/jni.cpp

```diff
@@ -208,12 +208,13 @@ bool jfieldIDWorkaround::is_valid_jfieldID(Klass* k, jfieldID id) {
 
 intptr_t jfieldIDWorkaround::encode_klass_hash(Klass* k, intptr_t offset) {
```

Comment:
The caller always passes an int, so why not change the parameter to int?
```suggestion
intptr_t jfieldIDWorkaround::encode_klass_hash(Klass* k, int offset) {
```